README.md

```diff
@@ -220,6 +220,11 @@ A governance platform where votes are not linked to Wallets. ZK can be used to f
 <br>
 REST APIs can log IP addresses of the data source and the receiver. Building a decentralized layer for enabling privacy preserving APIs alongside rate-limiting with RLN can be quite impactful</details>
 
+<details>
+<summary><b>Decentralized disaster communication</b></summary>
+<br>
+Decentralized network that allows for communication in disaster-stricken areas where traditional infrastructure may be unavailable or affected. Devices can connect directly to each other, forming a resilient network.
```

Comment:
by traditional infrastructure, do you mean the internet?
If so, can you add additional details about the communication medium here, so that the implementers may have some direction to work towards? i.e bluetooth, etc